### PR TITLE
misc(release): plug release registry

### PR DESCRIPTION
**Pull request by Chickensoupwithrice**
## Description

Plugs in the release registry

---

## Checklist

- [ ] Update [CHANGELOG.md](https://github.com/sourcegraph/sourcegraph/blob/main/CHANGELOG.md)
- [ ] Update [K8s Upgrade notes](https://github.com/sourcegraph/sourcegraph/blob/main/doc/admin/updates/kubernetes.md)
- [ ] Kustomiz-specific changes
- [ ] Update sister repository: [deploy-sourcegraph-helm](https://github.com/sourcegraph/deploy-sourcegraph-helm)
- [ ] Update sister repository: [deploy-sourcegraph-docker](https://github.com/sourcegraph/deploy-sourcegraph-docker)
- [ ] Verify all images have a valid tag and SHA256 sum

## Test plan

Assumed correct, copied from deploy-sourcegraph

### CODE REVIEW

**Comment by Chickensoupwithrice** [comment]
Waiting for testing before merge: https://github.com/sourcegraph/sourcegraph/issues/61078#issuecomment-2082883491

**Review by DaedalusG — APPROVED** [review]

**Pull request merged**
